packages/component-library/src/utils/formatters.test.js

```diff
@@ -0,0 +1,20 @@
+import { numeric, percentage, dollars } from './formatters';
+
+describe('formatters', () => {
+  it('should format numbers correctly', () => {
+    expect(numeric(0.1)).to.eql('0');
+    expect(numeric(0.5)).to.eql('1');
+    expect(numeric(1000)).to.eql('1,000');
+    expect(numeric(1000000)).to.eql('1 million');
+    expect(numeric(7300000)).to.eql('7.3 million');
+  });
```

Comment:
Might make sense to add tests for billions and trillions for the caveats mentioned above.